beryllium/src/main/java/de/b3/bubatz_service/rest/DepositoryController.java

```diff
@@ -0,0 +1,34 @@
+package de.b3.bubatz_service.rest;
+
+import de.b3.bubatz_service.articles.control.ArticleControl;
+import de.b3.bubatz_service.generated.api.DepositoryApi;
+import de.b3.bubatz_service.generated.models.*;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@CrossOrigin(origins = "*")
+@RestController
+public class DepositoryController implements DepositoryApi {
+
+    private final ArticleControl articleControl;
+
+    @Override
+    public ResponseEntity<List<GetArticle>> getArticles(State state) {
```

Comment:
state Query will be implemented in the next pr. Would be way to unreadable if done now

Reply:
nvm we dont need the query param in the current impl so it can be safely removed in this branch to prevent further merge conflicts. Would prob wait for the merge from #62 tho.